httpx/dispatch/http2.py

```diff
@@ -42,6 +44,7 @@ def __init__(
 
         self.events[stream_id] = []
         self.timeout_flags[stream_id] = TimeoutFlag()
+        self.window_update_received[stream_id] = asyncio.Event()
```

Comment:
I believe this be `self.backend.create_event`, is that right @florimondmanca?

Reply:
Correct! We should use methods on the backend instead of calling asyncio operations directly. :)